src/librustdoc/clean/mod.rs

```diff
@@ -2189,7 +2189,7 @@ impl Clean<Vec<Item>> for doctree::Import<'_> {
             });
         // Also check whether imports were asked to be inlined, in case we're trying to re-export a
         // crate in Rust 2018+
-        let please_inline = self.attrs.lists(sym::doc).has_word(sym::inline);
+        let please_inline = inlined;
```

Comment:
Rather than copying this, I would rename one or the other variables (maybe change `inlined` to `please_inline` for consistency).